orchagent/portsorch.cpp

```diff
@@ -4879,6 +4879,12 @@ bool PortsOrch::addBridgePort(Port &port)
         return true;
     }
 
+    if (port.m_rif_id != 0)
+    {
+        SWSS_LOG_NOTICE("Interface %s is a router port", port.m_alias.c_str());
```

Comment:
Can you please add more context in the log message and change it to warning since we are returning here in a failure scenario?
"Cannot create bridge port. Interface %s is a router port"

Reply:
Done.

Reply:
@liorghub  I recommend changing it to "WARN" log instead of notice since here are returning here in a failure scenario.

Reply:
@dgsudharsan this is actually a retry mechanism (not failure).
When we print this message we also return false in the next line, DB change is not consumed and we will try again till success.